### DISCUSSION TITLE
Calculating communication cost?

### DISCUSSION THREAD

**Discussion by Linardos** [FETS-AI/Challenge#85] [Q&A]
When I run any experiment, the resulting dataframe looks like this:
![image](https://user-images.githubusercontent.com/26694607/125259995-8d9c3400-e308-11eb-9c4b-991f18eb1abb.png)
My understanding is that time reflects the following, based on your instructions:
![image](https://user-images.githubusercontent.com/26694607/125260190-c0462c80-e308-11eb-8daf-f581ae40e6ec.png)
I am wondering whether I could break up this "time" into these 4 fundamental parts to figure out how much the communication cost is

**Comment by sarthakpati** [comment]
Nope, since this metric is integral during the performance evaluation of each submitted algorithm. You can add any metric written for PyTorch (for example, using [torchmetrics](https://torchmetrics.readthedocs.io/en/latest/references/modules.html#classification-metrics)) for validating your trained model.
